contrib/mercurial/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "mercurial"
+pkgver = "6.7.3"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+makedepends = [
+    "gettext-devel",
+    "musl-devel",
+    "python-devel",
+    "zlib-devel",
+    "zstd-devel",
+]
+checkdepends = ["python-pytest"]
```

Comment:
runtime dependencies are missing

Reply:
which?

Reply:
like... all of them

look at the package, it has none

```
=> mercurial-6.7.4-r0: running pre_pkg hook: 001_runtime_deps...
   SONAME: libc.so (provider: musl)
```

first of all, python

second of all, you add these makedepends like zlib/zstd and whatever but then the package contains no reference to them whatsoever

Reply:
i removed zlib/zstd and added python to depends